Python/symtable.c

```diff
@@ -1909,6 +1915,14 @@ symtable_visit_stmt(struct symtable *st, stmt_ty s)
                 return 0;
             }
         }
+
+        if (_PyAST_GetDocString(s->v.ClassDef.body)) {
```

Comment:
Do we have tests for these?

Reply:
Currently we have no public access to `ste_has_docstring`, I'll add internal C API tests for this.

Reply:
We don't necessarily need access to this field. The `__doc__` field should be correct.

Reply:
Add some naive tests for classes with/without docstring.